packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTree.tsx

```diff
@@ -137,13 +119,7 @@ export function ModelsTree(props: ModelsTreeProps) {
   );
 }
 
-const customizeTreeNodeItem = combineTreeNodeItemCustomizations([
-  addCustomTreeNodeItemLabelRenderer,
-  addTreeNodeItemCheckbox,
-  addModelsTreeNodeItemIcons,
-]);
-
-function useModelsTreeNodeLoader(props: ModelsTreeProps) {
+function useModelsTree({ filterInfo, onFilterApplied, ...props }: ModelsTreeProps) {
```

Comment:
A bit of a nitpick, but could `useModelsTree` and `useTree` have a `State` suffix to match `useVisibilityTreeState` and `usePresentationTreeState`?